charts/radar-s3-connector/templates/configmap.yaml

```diff
@@ -13,11 +13,20 @@ data:
     connector.class=io.confluent.connect.s3.S3SinkConnector
     tasks.max=4
     topics={{ .Values.topics }}
-    flush.size=80000
+    flush.size=1000
     s3.bucket.name={{ .Values.bucketName }}
     aws.access.key.id={{ .Values.bucketAccessKey }}
     aws.secret.access.key={{ .Values.bucketSecretKey }}
     store.url={{ .Values.s3Endpoint }}
     storage.class=io.confluent.connect.s3.storage.S3Storage
     format.class=org.radarbase.connect.s3.RadarBaseAvroFormat
+    rotate.interval.ms = 900000
```

Comment:
Is this `flush.size` and `rotate.interval.ms` because of earlier IOException you saw? I'd rather make the rotate interval smaller than the flush size: if multiple large records get uploaded it would be nice to add them all at once.

Reply:
Hmm, I thought flush size related to size in bytes, but it is the number of records. This might be an issue: for relatively frequent topics (e.g. 5 records per second) and 200 users, this would require 1 write per second. For high volume topics, with up to 1 MB per record, it will fill up the memory (not sure if it uses memory though or a local file cache?) It would be good to investigate if we can either increase the flush size again or separate topics that are high-volume and use a lower `flush.size` for those.

Reply:
@blootsvoets Yes. You are right. `flush.size` is for the records written in one file. The `flush.size` should be ideally higher and should increased when the studies start. Actually this is a wrong configuration. This configuration should be `rotate.schedule.interval.ms = 900000`. Again we need to reconsider the options when the studies start based on incoming traffic. The HDFS sink connector seems to work differently from s3-connector. HDFS connector was periodically committing offsets without problems and it would write the file when the flush size is met. S3 connector seems to be stuck after opening record writer when there is not enough records available for consuming. So I tried to set the rotate.interval.ms similar to what we had in HDFS connector. But s3 connector only allows `rotate.interval.ms` configuration when `TimeBasedPartitioner` is used. For DefaultPartitioner, we have the option to set `rotate.schedule.interval.ms = 900000`. But it would still commit the files, when the flush.size is not met. But the connector does not get stuck. I agree this is not suitable configuration for production yet. I would be happy to have a look at the configuration. I may need your ideas as well to figure out the most suitable configuration for s3 connector.

Reply:
We can change the `flush.size` anytime, I believe. The connector would reflect it after a restart.